src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -34,29 +33,11 @@ public static void validateLottoNumberRange(List<Integer> lottoNumbers) {
         }
     }
 
-    public static LottoTicket of(LottoTicketCreateStrategy strategy) {
-        return new LottoTicket(strategy.getLottoNumbers());
-    }
-
-    public int countSameNumber(List<Integer> target) {
-        return (int) target.stream()
-                .filter(this.lottoNumbers::contains)
-                .count();
-    }
-
-    public boolean isContainNumber(int number) {
-        return lottoNumbers.contains(number);
-    }
-
-    public Rank getRank(List<Integer> winningNumbers, int bonusBall) {
-        return Rank.of(countSameNumber(winningNumbers), isBonusWin(winningNumbers, bonusBall));
+    public Rank getRank(Win win) {
+        return win.getRank(this.lottoNumbers);
```

Comment:
`getLottoNumbers()` 를 사용하는게 좋을 것 같아요 :) 
```suggestion
        return win.getRank(this.getLottoNumbers);
```